src/store/albums.js

```diff
@@ -116,6 +123,14 @@ export default {
           return Promise.resolve(false);
         });
     },
+    async read({ commit, rootState }, id) {
```

Comment:
I don't think we want to mix async syntax and promise syntax in the same file.

Reply:
We already do this in a few files, for example: `src/api/fetch.js` and `src/store/tracks.js`
In time I would want to switch completely to async/await, but would not hold back using more modern syntax now in new additions/changes. It wil be easier to switch with gradual changes.

Reply:
I agree with switching to async await completely in time. I was not aware of any files mixing the two methods of doing things, but that doesn't mean we should start doing that more, since that's generally considered bad style.

Reply:
sure, I'll edit and create an issue for this switch